packages/runtime/src/useCases/consumption/attributes/CreateRepositoryAttribute.ts

```diff
@@ -39,6 +40,25 @@ export class CreateRepositoryAttributeUseCase extends UseCase<CreateRepositoryAt
                 ...request.content
             }
         });
+
+        const queryForRepositoryAttributeDuplicates = flattenObject({
+            content: {
+                "@type": "IdentityAttribute",
+                owner: this.accountController.identity.address.toString(),
+                value: request.content.value
+            }
+        });
+
+        queryForRepositoryAttributeDuplicates["succeededBy"] = { $exists: false };
+
+        const existingRepositoryAttributes = await this.attributeController.getLocalAttributes(queryForRepositoryAttributeDuplicates);
+
+        const filterForExactValueContent = existingRepositoryAttributes.some((duplicate) => _.isEqual(duplicate.content.value.toJSON(), request.content.value));
```

Comment:
I'm not too fond of the name which I would consider to be more appropriate for an actual filtering function rather than its result.
What about `exactMatchExists` ?